packages/graphics/libglvnd/package.mk

```diff
@@ -9,10 +9,11 @@ PKG_SITE="https://github.com/NVIDIA/libglvnd"
 PKG_URL="https://github.com/NVIDIA/libglvnd/archive/v${PKG_VERSION}.tar.gz"
 PKG_DEPENDS_TARGET="toolchain libX11 libXext xorgproto"
 PKG_LONGDESC="libglvnd is a vendor-neutral dispatch layer for arbitrating OpenGL API calls between multiple vendors."
-PKG_TOOLCHAIN="autotools"
+PKG_TOOLCHAIN="meson"
```

Comment:
Please remove PKG_TOOLCHAIN and allow auto-detect

Reply:
indeed, can be dropped. I'll push an update later